### PR TITLE
Feature/swagger

### PR DESCRIPTION
**Pull request by blabadi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +95,9 @@
     "multer": "^1.4.1",
     "node-fetch": "^2.6.0",
     "nodemailer": "^6.2.1",
-    "winston": "^3.2.1"
+    "swagger-ui-express": "^4.0.7",
+    "winston": "^3.2.1",
+    "yamljs": "^0.3.0"
```
Comment (andricDu) [inline package.json]: A whole dependency just because we can't save an openapi spec as json :stuck_out_tongue:

Comment (blabadi) [reply package.json]: haha you can save open api spec as json, but I opted for yaml #YAML_LIFE

Comment (andricDu) [reply package.json]: as npm dependencies go, this one is actually not awful.

**Comment by blabadi** [comment]
@joneubank  I have to merge this since it's blocking my next work, I added you to check the swagger file whenever you have time

**Pull request merged**